lib/build.rb

```diff
@@ -66,7 +69,10 @@ def query_branches
             login = branch_details.commit.author.login
           end
 
-          @potential_builds << PotentialBuild.new(@client, @token, @repository, nil, b.commit.sha, b.name, login, nil, nil, nil, nil, nil)
+          associated_pr_num = @branch_to_pr_number.fetch(b.name, nil)
```

Comment:
The PR check should have been called first, which would have populated this hash.

These lines simply:
 - Get the PR number for this branch, or nil
 - Create a new PotentialBuild instance as before
 - Call a method to assign the pr number to the potential build instance
 - Add the potential build to the list of potential builds